README.md

```diff
@@ -95,8 +95,8 @@ Your code should defend against [edge cases](http://programmers.stackexchange.co
 | Airport | airportPlanes[@Plane]     | landPlane(@Plane) | @Void   |
 | Plane   | id @String                |                   |         |
 
-// Test 1 - Checks that landPlane instructs airport to land a Plane
-// Test 2 - Tests that method landPlane only works with a plane
+**Test 1** - Checks that landPlane instructs airport to land a Plane
+**Test 2** - Tests that method landPlane only works with a plane
 
```

Comment:
I was unsure whether the method landPlane should, apart from instructing the airport to land a plane, also add a plane into the array airportPlanes. If that's the case, which is what I've done, does the method fulfil the single responsibility principle? A single method is doing two things: adding a plane to the array, and instructing the airport to land a plane.